### PR TITLE
Drop a volatile CP

### PR DESCRIPTION
**Pull request by mixonic**
Volatile CPs are deprecated in Ember 3.8:
https://deprecations.emberjs.com/v3.x/#toc_computed-property-volatile

### CODE REVIEW

**Comment by mixonic** [comment]
I've got some legit looking failure here to consider.

File: addon/components/sortable-group.js [hunk 1]
```diff
@@ -33,17 +33,10 @@ export default Component.extend({
...
-    Position for the first item.
-    If spacing is present, first item's position will have to change as well.
-    @property itemPosition
-    @type Number
-  */
-  itemPosition: computed(function() {
-    let direction = this.get('direction');
-
-    return this.get(`sortedItems.firstObject.${direction}`) - this.get('sortedItems.firstObject.spacing');
-  }).volatile(),
```
Comment (jgwhite) [inline addon/components/sortable-group.js]: I don’t remember why this property needed to be volatile. At first glance, it looks like we could just declare the dependent keys as `sortedItems.firstObject.{x,y,spacing}` but maybe I’m missing something.

**Comment by btecu** [comment]
What is the status of this?

**Comment by jgwhite** [comment]
I’ve not checked up on it since April. @mixonic do you want to pursue this PR any further in its current state?

**Comment by btecu** [comment]
I was asking because I'm getting deprecation warnings about `volatile`.

**Comment by jgwhite** [comment]
@btecu 100%

We could try reformulating this PR along these lines: https://github.com/heroku/ember-sortable/pull/246/files#r278399195

**Comment by dbashford** [comment]
Any updates looming here?

**Comment by mixonic** [comment]
It is a fun puzzle to get this fixed, and eventually I'll need to use this lib with Ember 3.8 (I hope) but I am not prioritizing this at work. I'll close it to reflect that it is not actively "in progress". Branch will remain available.

**Pull request closed without merging**